desktop/src/main/java/bisq/desktop/components/MenuItem.java

```diff
@@ -58,7 +58,7 @@ public MenuItem(Navigation navigation,
         this.viewClass = viewClass;
         this.baseNavPath = baseNavPath;
 
-        setLabelText(title);
+        setLabelText(title.toUpperCase());
```

Comment:
This doesn't handle the 'Vote on proposals'/'Browse open proposals' text that changes depending on phase

Reply:
Could be fixed in GovernanceView.java:90/92

Reply:
Thanks for pointing it out. Just fixed it by converting to uppercase also on label update.

Reply:
And I removed now an unnecessary uppercasing. Should have looked through the code more carefully 🙄

Reply:
Yes, better to only set it inside setLabelText as you do now